deal.II-toolchain/packages/petsc.package

```diff
@@ -80,10 +80,27 @@ if [ ! -z "${PARMETIS_DIR}" ]; then
         ${CONFOPTS} \
         --with-parmetis-dir=${PARMETIS_DIR} \
 	--with-metis-dir=${PARMETIS_DIR}"
+fi
+
+if [ ! -z "${SCALAPACK_DIR}" ]; then
+    cecho ${INFO} "PETSc: configuration with SCALAPACK"
+    CONFOPTS="\
+        ${CONFOPTS} \
+        --with-scalapack-dir=${SCALAPACK_DIR}"
+else
+  if [ ! -z "${PARMETIS_DIR}" ]; then
+    CONFOPTS="${CONFOPTS} --download-scalapack=1"
+  fi
+fi
 
-    for external_pkg in scalapack mumps; do
-        CONFOPTS="${CONFOPTS} --download-${external_pkg}=1"
-    done
+if [ ! -z "${MUMPS_DIR}" ]; then
+    cecho ${INFO} "PETSc: configuration with MUMPS"
+    CONFOPTS="${CONFOPTS} \
+        --with-mumps-dir=${MUMPS_DIR}"
+else
+  if [ ! -z "${PARMETIS_DIR}" ]; then
+    CONFOPTS="${CONFOPTS} --download-mumps=1"
+  fi
```

Comment:
Why do we need to check for parmetis?

Reply:
Same answer as above: I have not tested whether this check is necessary, I just assumed that it is necessary because it was already there.